src/cpp_bindgen/generator.cpp

```diff
@@ -154,6 +154,7 @@ namespace cpp_bindgen {
     void generate_fortran_interface(std::ostream &strm, std::string const &module_name) {
         strm << "! This file is generated!\n";
         strm << "module " << module_name << "\n";
+        strm << "use iso_c_binding\n"; // pgi wants to have this
```

Comment:
would this be needed only for PGI or is it good for all the other compilers, too?

Reply:
Pgi wants it, all others don’t care.